### DISCUSSION TITLE
Notifiche richieste di uscita anticipata

### DISCUSSION THREAD

**Discussion by Forrest-It** [iisgiua/giuaschool#428] [Richieste di aiuto]
Buon pomeriggio ,

volevo chiedere se è possibile abilitare una notifica ai destinatari di una richiesta di uscita anticipata .

Aggiungo, inoltre, che dal ruolo "Staff" -> Studenti > Richieste non sono in lista le richieste di uscita anticipata : è normale o è un baco ?

Grazie

**Comment by trinko** [comment]
Al momento c'è una sorta di notifica, solo per lo staff, tramite avviso nella home.
Comunque la segno come richiesta.

**Comment by Forrest-It** [comment]
Grazie mille

**Comment by ghost** [comment]
@Forrest-It Buongiorno! Avrei bisogno di parlarti, in merito a questa richiesta. Posso parlarti in privato? Mi scrivi alla mia mail: padovesi.roberto@gmail.com? Grazie!
